src/create_disk.sh

```diff
@@ -67,14 +68,25 @@ else
 	ext="AA64"
 fi
 
+# ensure that home is created with the correct SELinux context
+realvar=$(readlink -f rootfs/ostree/deploy/*/var)
+mkdir -p "${realvar}/home"
+setfattr -n "security.selinux system_u:object_r:home_root_t:so" "${realvar}/home"
```

Comment:
Hmm, I think this should be `-n security.selinux -v ...`, and `s/so/s0/`, right? Also, don't we want other things in there, notably `/var/lib` and `/var/log` at least (see https://github.com/coreos/ignition-dracut/pull/79#issuecomment-488446949)?

Let's make it conditional on the same `save-var-subdirs-for-selabel-workaround` flag we made for this in the Anaconda path? (Although `save` is a little inaccurate now since we're actively creating them rather than sparing them from deletion).

Hardcoding specific labels is OK, though you should be able to use `setfiles -v rootfs/etc/selinux/targeted/contexts/files/file_contexts /var` instead.

Reply:
The last bit should be `s0` and not `so`.